index.html

```diff
@@ -45,7 +45,7 @@ <h2 data-i18n="app.title"></h2>
                             <input class='col8' id='address' placeholder='' data-i18n="[placeholder]step1.addr"/>
                             <button class='col4' id='findme' data-i18n="step1.button"></button>
                             <br />
-                            Or <a id="use_my_location" href="#" onclick="return false">use my current location</a>.
+                            <a id="use_my_location" data-i18n="step1.use_location" href="#" onclick="return false">use my current location</a>.
```

Comment:
I did see the italian version ended up up using https://github.com/domoritz/leaflet-locatecontrol after this PR, I might revisit it a bit later to move us to that if the 'from facebook' or similar extra tools don't make it in.